src/test/mir-opt/array_index_is_temporary.main.SimplifyCfg-elaborate-drops.after.mir.32bit

```diff
@@ -0,0 +1,97 @@
+// MIR for `main` after SimplifyCfg-elaborate-drops
```

Comment:
This file is almost a duplicate of another file, the only difference is the `_` vs `-` in the main name

Reply:
i moved these by hand so there's probably a couple errors hiding still.